Automattic/wp-calypso
# Add default values, placesholder and id to form fields that apply

#### Changes proposed in this Pull Request
This PR adds some missing attributes that are currently supported by the field shortcode. 

The other one that is still missing is the values (so that we can specify values for (multi checkboxes, 
radio and drop down selection) I think that could be done in a different PR. Since the UI is not something that is 100% clear yet.

Textarea Field
![screen_shot_2018-11-15_at_12_21_53_pm](https://user-images.githubusercontent.com/115071/48583336-b069b680-e8db-11e8-99b3-c3db23e0cb6f.png)

Email, Name, Text, Phone, Date, URL field
![screen_shot_2018-11-15_at_12_22_01_pm](https://user-images.githubusercontent.com/115071/48583379-d1320c00-e8db-11e8-9d5c-6ab233de3e5b.png)

Multi Checkbox, Radio and Select Field
![screen_shot_2018-11-15_at_12_22_09_pm](https://user-images.githubusercontent.com/115071/48583430-f32b8e80-e8db-11e8-9cc6-88431327428c.png)

Checkbox
![screen_shot_2018-11-15_at_12_22_27_pm](https://user-images.githubusercontent.com/115071/48583608-66cd9b80-e8dc-11e8-989f-7a3925060797.png)

#### Testing instructions

1. Check out this Jurassic Ninja link.
    https://jurassic.ninja/create/?gutenberg&gutenpack&shortlived&jetpack-beta&calypsobranch=update/contact-form-block-icon&branch=add/contact-form-gutenblock-sdk

2. Connect jetpack

3. Start a new page

4. Add a the new contact form block

5. Are you able to edit the placeholder and default values as expected.


Fixes #
